<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: QLinearGradient(x1: 1, y1: 0, 
                            x2: 0, y2: 1, 
                            stop: 0   rgba(168, 214, 252, 255), 
                            stop: 0.5 rgba(217, 184, 255, 255), 
                            stop: 1   rgba(177, 110, 197, 255));</string>
  </property>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Montserrat';
font-style: normal;
font-weight: 700;
font-size: 35px;
line-height: 122px;
color: #000;
background: none;</string>
   </property>
   <property name="text">
    <string>strings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="print_register_btn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>420</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 background-color: #62B1FF;
	border-radius: 20%;
	font-family: 'Montserrat';
	font-style: normal;
	font-weight: 400;
	font-size: 15px;
	color: #000000;
	border: 1px solid rgba(255, 255, 255, 0.01);
}
QPushButton:hover{
   background-color:#7DBFFF;
}
</string>
   </property>
   <property name="text">
    <string>print register</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="info_test">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
background: #FFFFFF;
border-radius: 20%;
padding: 13px 17px;
font-family: 'Montserrat';
font-style: normal;
font-weight: 400;
font-size: 13px;
line-height: 16px;
border: 1px solid rgba(255, 255, 255, 0.01);
}
QTextEdit:focus{
	border: 1px solid #000;
}
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:13px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_register_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 background-color: #62B1FF;
	border-radius: 20%;
	font-family: 'Montserrat';
	font-style: normal;
	font-weight: 400;
	font-size: 15px;
	color: #000000;
border: 1px solid rgba(255, 255, 255, 0.01);
}
QPushButton:hover{
   background-color:#7DBFFF;
}



</string>
   </property>
   <property name="text">
    <string>delete register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_register_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 background-color: #62B1FF;
	border-radius: 20%;
	font-family: 'Montserrat';
	font-style: normal;
	font-weight: 400;
	font-size: 15px;
	color: #000000;
	border: 1px solid rgba(255, 255, 255, 0.01);
}
QPushButton:hover{
   background-color:#7DBFFF;
}




</string>
   </property>
   <property name="text">
    <string>update register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_field_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background: #FFFFFF;
border-radius: 20%;
padding: 13px 17px;
font-family: 'Montserrat';
font-style: normal;
font-weight: 400;
font-size: 13px;
line-height: 16px;
border: 1px solid rgba(255, 255, 255, 0.01);
}
QLineEdit:focus{
	border: 1px solid #000;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_field_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background: #FFFFFF;
border-radius: 20%;
padding: 13px 17px;
font-family: 'Montserrat';
font-style: normal;
font-weight: 400;
font-size: 13px;
line-height: 16px;
border: 1px solid rgba(255, 255, 255, 0.01);
}
QLineEdit:focus{
	border: 1px solid #000;
}
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
